tests/dialects/test_databricks.py

```diff
@@ -38,6 +38,8 @@ def test_databricks(self):
             "CREATE FUNCTION add_one(x INT) RETURNS INT LANGUAGE PYTHON AS $FOO$def add_one(x):\n  return x+1$FOO$"
         )
 
+        self.validate_identity("TRUNCATE TABLE t1 PARTITION(age = 10, name = 'test', address)")
```

Comment:
Let's also add a test with the `LIKE` operator shown in Databricks' docs

Reply:
Done